.github/workflows/ci.yml

```diff
@@ -3,18 +3,19 @@ on: [push, pull_request]
 jobs:
   tests:
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        node: [18, 20]
+      coninue-on-error: ${{ matrix.node == 20 }}
```

Comment:
```suggestion
```
Since 2nd step to upgrade to v20 was already cleared before this, we need to remove `continue-on-error` in order to make the Node 20 test a blocking one.